### PR TITLE
Kaining add description

### PR DESCRIPTION
**Pull request by Kylin-Mao**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

## Note:
Include the information the reviewers need to know.

### CODE REVIEW

**Review by Yanglin-Tao — COMMENTED** [review]
Is task title guaranteed to be non-empty? Any checks necessary?

**Comment by Kylin-Mao** [comment]
> Is task title guaranteed to be non-empty? Any checks necessary?
Thanks for reminding. Modified it.

File: src/open_source_python_template/crawlTasks.py [hunk 1]
```diff
@@ -18,7 +18,9 @@ def get_tasks():
         if creds and creds.expired and creds.refresh_token:
             creds.refresh(Request())
         else:
-            flow = InstalledAppFlow.from_client_secrets_file("credential.json", SCOPES)
+            flow = InstalledAppFlow.from_client_secrets_file(
+                "src/open_source_python_template/credential.json", SCOPES
```
Comment (zwinniez) [inline src/open_source_python_template/crawlTasks.py:22]: is there a reason why we have the entire path as opposed to the relative path here? The import didn't work for me with the full path

File: src/open_source_python_template/crawlTasks.py [hunk 2]
```diff
@@ -38,10 +40,15 @@ def get_tasks():
...
                     {
-                        "title": task["title"],
+                        "title": task["title"] if "title" in task else "No title",
                         "id": task["id"],
                         "tasklist_id": tasklist["id"],
                         "tasklist_title": tasklist["title"],
+                        "taskdescription": (
+                            task["notes"]
+                            if "notes" in task
+                            else "No description available"
```
Comment (zwinniez) [inline src/open_source_python_template/crawlTasks.py:50]: thanks for catching up - this update on webpage.

File: src/open_source_python_template/crawlTasks.py (hunk 1 above)
Comment (Kylin-Mao) [reply src/open_source_python_template/crawlTasks.py:22]: That‘s kinda wired because when I do not use the entire path it doesn't work for me.

Comment (zwinniez) [reply src/open_source_python_template/crawlTasks.py:22]: resolved, thanks!

**Review by zwinniez — APPROVED** [review]

**Pull request merged**